concent_api/verifier/utils.py

```diff
@@ -193,7 +196,7 @@ def compare_minimum_ssim_with_results(ssim_list, subtask_id):
         raise VerificationMismatch(subtask_id=subtask_id)
 
 
-def load_images(blender_output_file_name, result_file, subtask_id):
+def load_images(blender_output_file_name: str, result_file: str, subtask_id: str) -> Union[ndarray, ndarray]:
```

Comment:
I think `Union` means that you return either one or another value. I think it should be `tuple` in this case.